Adding ios_l2_interfaces model in new format (#9)
ansible-network/resource_module_models: ios/l2_interfaces/deleted_example_01.txt

# Using Deleted

# Before state:
# -------------
#

> # viosl2#show interfaces switchport

Reviewer: IMHO, showing the running configuration for the interfaces should be enough here ( and below)

Author: @cidrblock have updated the before n after state e.g. to use `show running-config | section ^interface` instead of `show interfaces switchport`.

# Using Deleted

# Before state:
# -------------
#
# viosl2#show running-config | section ^interface
# interface GigabitEthernet0/1
#  description Configured by Ansible
#  switchport access vlan 20
#  switchport mode access
#  negotiation auto
# interface GigabitEthernet0/2
#  description This is test
#  switchport access vlan 20
#  switchport trunk encapsulation dot1q
#  switchport trunk native vlan 10
#  switchport mode trunk
#  media-type rj45
#  negotiation auto

- name: Delete IOS L2 interfaces as in given arguments
  ios_interfaces:
    config:
      - name: GigabitEthernet0/1
      - name: GigabitEthernet0/2

>     operation: deleted

Reviewer 2: ```suggestion
    state: deleted
```

# Using Deleted

# Before state:
# -------------
#
# viosl2#show running-config | section ^interface
# interface GigabitEthernet0/1
#  description Configured by Ansible
#  switchport access vlan 20
#  switchport mode access
#  negotiation auto
# interface GigabitEthernet0/2
#  description This is test
#  switchport access vlan 20
#  switchport trunk allowed vlan 20,40,60,80
#  switchport trunk encapsulation dot1q
#  switchport trunk native vlan 10
#  switchport trunk pruning vlan 10
#  switchport mode trunk
#  media-type rj45
#  negotiation auto

> - name: Delete IOS L2 interfaces as in given arguments

Reviewer 3: ```suggestion
- name: Delete L2 attributes of given interfaces (Note: This won't delete the interface itself)
```

>   ios_interfaces:

Reviewer 3: ```suggestion
  ios_l2_interfaces:
```
